frontend/src/main/components/SectionsTableBase.js

```diff
@@ -37,19 +37,20 @@ export default function SectionsTableBase({ columns, data, testid = "testid"}) {
                     <td
                     {...cell.getCellProps()}
                     data-testid={`${testid}-cell-row-${cell.row.index}-col-${cell.column.id}`}
-                    // Stryker disable next-line ObjectLiteral
+                    // Stryker disable all
                     style={{background: cell.isGrouped ? "#e5fcf4" : cell.isAggregated ? "#e5fcf4" : "#effcf8", fontWeight: cell.isGrouped ? "bold" : cell.isAggregated ? "bold" : "normal"}}
                     >
-                    
                     {cell.isGrouped ? (
                     <>
                     <span {...row.getToggleRowExpandedProps()}
-                    data-testid={`${testid}-cell-row-${cell.row.index}-col-${cell.column.id}-expand-symbols`}
-                    >
-                    {row.isExpanded ? "➖ " : "➕ "}
-                    </span>{" "}
+                    data-testid={`${testid}-cell-row-${cell.row.index}-col-${cell.column.id}-expand-symbols`} 
+                    > 
+                    {row.subRows.length > 1 ? row.isExpanded ? "➖ " : "➕ " : null} 
```

Comment:
I like the use of the ? operator.  I wonder, though, if this would be more clear to future programmers if we included some (redundant) parens to show the operator order?  Something like this:
```suggestion
                    {  ( row.subRows.length > 1  ) ? ( row.isExpanded ? "➖ " : "➕ " ) : null} 
```